chapters/projectManagement.typ

```diff
@@ -5,9 +5,10 @@
 // [x] Zeiterfassung (Stunden pro Woche/Stunden pro Task-Kategorie, wie Implementation Doku, Meeting, etc.)
 // Hinweis: Keine Liniendiagramme für die Darstellung von Zeit/Arbeitsaufwand pro Woche
 
-@approach explains how the project was approached.
-@project-plan looks into the planning of the project, including a comparative analysis between the initial plan and its effective implementation.
-Finallly, @time-tracking provides a summarized overview of the allocated working hours coupled with a reflective assessment of the time invested.
+The approach of the project is explained in @approach.
```

Comment:
Ez isch halt alles wieder ide vergangeheit form..

Reply:
hmm nee isch doch "is explained", ned "was explained"